addons/html_builder/static/tests/builder_action.test.js

```diff
@@ -122,4 +122,62 @@ describe("HTML builder tests", () => {
         await animationFrame();
         expect.verifySteps(["prepare"]);
     });
+
+    describe("reload", () => {
+        test("reload: iframe is reloaded after apply", async () => {
+            addBuilderAction({
+                customAction: {
+                    reload: {},
+                    apply: () => {
+                        expect.step("apply");
+                    },
+                },
+            });
+            addBuilderOption({
+                selector: ".s_test",
+                template: xml`<BuilderButton action="'customAction'">Click</BuilderButton>`,
+            });
+            const def = new Deferred();
+            const { contentEl } = await setupHTMLBuilder(`<section class="s_test">Test</section>`);
+            const contentDocument = contentEl.ownerDocument.documentElement;
+            const iframe = contentEl.ownerDocument.defaultView.frameElement;
+            await contains(":iframe .s_test").click();
+            await contains("[data-action-id='customAction']").click();
+            expect.verifySteps(["apply"]);
+            // Make sure the iframe had the time to reload.
+            setTimeout(() => {
+                def.resolve();
+            }, 50);
+            await def;
```

Comment:
It s not a good practice to use a setTimeout. It s not possible to check something in the dom ?